### PR TITLE
Adding android bottom navigate 

### PR DESCRIPTION
**Pull request by eliasreis54**
navigate

About IOS, i found `CupertinoTabBar`, but i'm not sure if it's right.

Is this the correct widget to use on IOS?

If yes, i start to build it.

### CODE REVIEW

File: lib/pages/feed/AuthorInsights.dart [hunk 1]
```diff
@@ -46,17 +46,13 @@ class AuthorInsights extends StatelessWidget {
           Row(children: [
             Container(
               margin: EdgeInsets.all(5),
-              child: Column(children: [
-                commentAsset,
-                Text(commentsCount.toString())
-              ]),
+              child: Column(
+                  children: [commentAsset, Text(commentsCount.toString())]),
```
Comment (mugbug) [inline lib/pages/feed/AuthorInsights.dart]: Prefer always adding commas after parentheses, brackets etc so the text editor knows how to format your code.

File: lib/pages/home.dart [hunk 2]
```diff
@@ -2,35 +2,69 @@ import 'package:flutter/widgets.dart';
...
+
+  @override
+  void initState() {
+    super.initState();
+    _currentIndex = 0;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    setState(() => {_context = context});
```
Comment (mugbug) [inline lib/pages/home.dart]: Hm, I'm not sure if this is a good idea. @erickzanardo what do you think, flutter master?

File: lib/pages/home.dart [hunk 3]
```diff
@@ -2,35 +2,69 @@ import 'package:flutter/widgets.dart';
...
+
+class _HomeState extends State<Home> {
+  static BuildContext _context;
+  static int _currentIndex;
+
+  final List<Widget> _pages = [
+    PixelList(),
+    Text('search'),
+    SelectPixel(),
+    Center(
```
Comment (mugbug) [inline lib/pages/home.dart]: Move this code to another widget to keep code clean

File: lib/pages/feed/AuthorInsights.dart (hunk 1 above)
Comment (eliasreis54) [reply lib/pages/feed/AuthorInsights.dart]: :+1:
Done

File: lib/pages/home.dart (hunk 3 above)
Comment (eliasreis54) [reply lib/pages/home.dart]: :+1:
Done

File: lib/pages/home.dart (hunk 2 above)
Comment (eliasreis54) [reply lib/pages/home.dart]: After remove some codes, it's not necessary anymore.

**Review by mugbug — APPROVED** [review]

File: lib/widgets/bottom_navigate.dart [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+final Widget gridAssetWidget =
+    SvgPicture.asset(gridAsset, semanticsLabel: 'Grid pixel');
+final Widget profileAssetWidget =
+    SvgPicture.asset(profileAsset, semanticsLabel: 'Profile');
+
+class BottomNavigate {
+  final int currentIndex;
+  Function onTap;
+
+  BottomNavigate({@required this.currentIndex, @required this.onTap});
```
Comment (erickzanardo) [inline lib/widgets/bottom_navigate.dart:46]: Any reason why this is not a widget itself, with a build method and so?

**Review by erickzanardo — APPROVED** [review]
LGTM

File: lib/widgets/bottom_navigate.dart (hunk 4 above)
Comment (eliasreis54) [reply lib/widgets/bottom_navigate.dart:46]: I thought some difference was needed
between android and IOS.
But we don't need

Comment (erickzanardo) [reply lib/widgets/bottom_navigate.dart:46]: I think we should make this a Widget then, making Widget Factories is king of an anti pattern.

Comment (eliasreis54) [reply lib/widgets/bottom_navigate.dart:46]: Done!

**Pull request merged**